aldryn_search/router.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from django.conf import settings
-from django.utils.translation import get_language
+from cms.utils.i18n import get_current_language
```

Comment:
instead of doing this please import ```alias_from_language``` from ```.utils```, then do:

```python
language = get_language()
alias = alias_from_language(language)

if alias not in settings.HAYSTACK_CONNECTIONS:
    return DEFAULT_ALIAS
return alias
```